react/features/base/media/components/AbstractVideoTrack.js

```diff
@@ -11,7 +11,7 @@ import { Video } from './_';
 /**
  * The type of the React {@code Component} props of {@link AbstractVideoTrack}.
  */
-export type Props = {
+export type Props = {|
```

Comment:
What is this syntax?

Reply:
Exact object types you can check it out here - https://flow.org/en/docs/types/objects/ . Basically with the normal syntax you can pass additional properties that are not defined in the type and flow won't complain. Which I actually don't mind so much and wouldn't change the syntax but flow required it at some point during my changes...

Reply:
Actually when I moved the event handlers into `eventHandlers` prop I noticed that flow doesn't complain anymore about this. I can remove this syntax now!